supervisord.conf

```diff
@@ -0,0 +1,32 @@
+[supervisord]
+nodaemon=true
```

Comment:
regarding line that you removed from here. Could you bring line back? I asked what does this line?

I want you to understand what does your code. Not just simply copy from somewhere.

Reply:
Well, about `nodaemon` it will prevent supervisord from running in the background (as a daemon). Therefore, supervisord will be explicitly visible within Docker.

---
In latest commit I removed these lines as the supervisord's defaults are enough.

```
startsecs = 0 
startretries = 1
autorestart = true
```


```startsecs``` defaults to 0. It serves to indicate how much time a program must be running in order to be considered as succesful.

```startretries``` defatuls to 3. It serves to indicate how many run retries a failed program will have.

```autorestart``` defaults to unknown. It serves to restart a program that get an unexpected exit code, so in order to be useful you have to explicitly indicate the possible error codes that your program will have.